src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine [] machines = new Machine[]{new Truck(), new Excavator(), new Bulldozer()};
+        for (Machine machine: machines) {
```

Comment:
Enhanced for-loop spacing:
```suggestion
        for (Machine machine : machines) {
```